guides/v3.3.0/object-model/computed-properties.md

```diff
@@ -80,31 +80,68 @@ ironMan.fullName; // no console output since dependencies have not changed
 
 In the previous example, the `fullName` computed property depends on two other properties of the same object.  
 However, you may find that you have to observe the properties of a different object.
+
 For example, look at this computed property:
 
 ```javascript
 import EmberObject, { computed } from '@ember/object';
 
-let obj = EmberObject.extend({
-  baz: { foo: 'BLAMMO', bar: 'BLAZORZ' },
+let home = EmberObject.extend({
+  location: {
+    street: 'Evergreen Terrace',
+    nr: 742
+  }
+
+  address: computed('location.street', 'location.nr', function() {
+    return `${this.location.nr} ${this.location.street}`;
+  })
+});
+
+home.address // 742 Evergreen Terrace
+home.set('location.nr', 744)
+home.address // 744 Evergreen Terrace
+```
+
+Important to remember here is that you have to observe the object properties. Adding the object reference `'location'` as dependents key will not recalculate the computed property `address` when the `street` or `nr` properties change. It will only recalculate when the object reference `location` is updated.
```

Comment:
"It is important to observe an object's properties, not the object itself that has properties nested inside. If the object reference `'location'` is used as a dependent key, the computed property will not recalculate when the `street` or `nr` properties change."

I would remove "It will only recalculate when the object reference `location` is updated." I think this would just confuse beginners.

Reply:
I might come back to this phrasing later. I feel like my revision is still too complex.